src/main/java/uk/gov/hmcts/reform/pip/account/management/service/CustomAccountRetrievalService.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.pip.account.management.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.pip.account.management.database.UserRepository;
+import uk.gov.hmcts.reform.pip.account.management.errorhandling.exceptions.NotFoundException;
+import uk.gov.hmcts.reform.pip.account.management.helpers.EmailHelper;
+import uk.gov.hmcts.reform.pip.account.management.model.PiUser;
+import uk.gov.hmcts.reform.pip.account.management.model.Roles;
+import uk.gov.hmcts.reform.pip.account.management.model.UserProvenances;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.ALL_NON_RESTRICTED_ADMIN_ROLES;
+import static uk.gov.hmcts.reform.pip.account.management.model.Roles.ALL_NON_THIRD_PARTY_ROLES;
+import static uk.gov.hmcts.reform.pip.account.management.model.UserProvenances.ALL_NON_THIRD_PARTY_PROVENANCES;
+
+@Service
+public class CustomAccountRetrievalService {
```

Comment:
Similar comment to the controller name

Reply:
This one is renamed to AccountFilteringService :)